src/grt/src/fastroute/src/maze.cpp

```diff
@@ -1268,6 +1268,24 @@ void FastRouteCore::reInitTree(const int netID)
   checkAndFixEmbeddedTree(netID);
 }
 
+float getCost(const int i,
```

Comment:
When h_cost_table_ and v_cost_table_ are being built they should use this function as well to avoid code duplication.